CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
+
+## Version 1.2
+
+- TrackDependency will produce valid JSON when not all required fields were specified.
+- Redundant property ```RequestTelemetry.ID``` is now just a proxy for ```RequestTelemetry.Operation.Id```.
```

Comment:
I should have spelled it out as vNext so we can put the version later. Will fix it